CMakeLists.txt

```diff
@@ -2,6 +2,14 @@ cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
 project(gloo CXX C)
 
+set(GLOO_VERSION_MAJOR 0)
+set(GLOO_VERSION_MINOR 5)
+set(GLOO_VERSION_PATCH 0)
+set(GLOO_VERSION
+    "${GLOO_VERSION_MAJOR}.${GLOO_VERSION_MINOR}.${GLOO_VERSION_PATCH}")
+set(GLOO_VERSION_STRING ${GLOO_VERSION})
```

Comment:
Seems unused.

Reply:
Nope. Some cmake libraries like to use VERSION_STRING instead of VERSION, so I kind of kept it here. Shall we remove it? :)

Reply:
Either way is fine w/ me, but if we keep it, it should have a comment or something, otherwise somebody else will recognize it is unused and remove it :)